subsys/bluetooth/mesh/rpr_srv.c

```diff
@@ -232,11 +225,13 @@ static void scan_report_send(void)
 			continue;
 		}
 
-		BT_MESH_MODEL_BUF_DEFINE(buf, RPR_OP_SCAN_REPORT, 23);
-		bt_mesh_model_msg_init(&buf, RPR_OP_SCAN_REPORT);
-		net_buf_simple_add_u8(&buf, dev->rssi);
-		net_buf_simple_add_mem(&buf, dev->uuid, 16);
-		net_buf_simple_add_le16(&buf, dev->oob);
+		#define UUID_LIST(i, _) (dev->uuid[i])
```

Comment:
Is it a good idea to declare definition within function?

Reply:
Done.

Reply:
BTW: In fact, for the initialization of the array, using functions will save more code size, but the approach here will make the code look more concise :).